From c229c01a5821276de9ee275ccfebe99b51a1a8c8 Mon Sep 17 00:00:00 2001
From: JoeLametta <JoeLametta@users.noreply.github.com>
Date: Sat, 28 Nov 2020 17:44:21 +0000
Subject: [PATCH] Replace 'freedb.dbpoweramp.com' CDDB server with gnudb.org

It seems gnudb.org allows submissions too while 'freedb.dbpoweramp.com' is read only.

Signed-off-by: JoeLametta <JoeLametta@users.noreply.github.com>
---
 whipper/common/program.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/whipper/common/program.py b/whipper/common/program.py
index 97edd943..04e979e8 100644
--- a/whipper/common/program.py
+++ b/whipper/common/program.py
@@ -230,7 +230,7 @@ def getPath(self, outdir, template, mbdiscid, metadata, track_number=None):
     @staticmethod
     def getCDDB(cddbdiscid):
         """
-        Fetch basic metadata from dBpoweramp's mirror of freedb's CDDB.
+        Fetch basic metadata from gnudb.org's mirror of freedb's CDDB.
 
         Freedb's official CDDB isn't used anymore because it's going to be
         shut down on 31/03/2020.
@@ -244,7 +244,7 @@ def getCDDB(cddbdiscid):
         # FIXME: convert to nonblocking?
         try:
             md = freedb.perform_lookup(
-                     cddbdiscid, 'freedb.dbpoweramp.com', 80
+                     cddbdiscid, 'gnudb.gnudb.org', 80
             )
             logger.debug('CDDB query result: %r', md)
             return [item['DTITLE'] for item in md if 'DTITLE' in item] or None
